UsefullExtensionLib/usefullExtensionLibrary/src/main/java/com/nbprog/usefullextensionlibrary/StringExtension.kt

```diff
@@ -0,0 +1,19 @@
+package com.nbprog.usefullextensionlibrary
+
+val String.containsLatinLetter: Boolean
+    get() = matches(Regex(".*[A-Za-z].*"))
+
+val String.containsDigit: Boolean
+    get() = matches(Regex(".*[0-9].*"))
+
+val String.isAlphanumeric: Boolean
+    get() = matches(Regex("[A-Za-z0-9]*"))
+
+val String.hasLettersAndDigits: Boolean
```

Comment:
Czy to nie jest to samo co funkcja isAlphanumeric?

Reply:
done